uber/decorators.py

```diff
@@ -26,7 +26,12 @@ def with_check(*args, **kwargs):
                 pass  # we don't care about unrestricted pages for this version
             else:
                 sa.Tracking.track_pageview(cherrypy.request.path_info, cherrypy.request.query_string)
-        return func(*args, **kwargs)
+        try:
```

Comment:
You put this under `log_pageview` but I think you meant to put it in `renderable`. I almost didn't realize it.

If this wasn't an accident, note that the `@log_pageview` decorator is NOT applied to all pages -- in fact, we log relatively few types of page views.

Let me know if you want me to move the function, I'll be happy to stick it at the end of `renderable` instead.